
##glmapping parameter
#Awareness map radius d_Rho*n_Rho
#devition of orientation 360/d_Phi_deg
#n_Z = Z_below + Z_over + 1;
#Number of grid n_Rho*(360/d_Phi)*(nz_below+nz_over+1)

mlmapping_am_d_Rho:         0.1
mlmapping_am_d_Phi_deg:       2
mlmapping_am_d_Z:           0.1
mlmapping_am_n_Rho:          30
mlmapping_am_n_Z_below:      20
mlmapping_am_n_Z_over:        5

#Lolbalmap papameter
#n_xy/n_z available value     3x(2n+1)  3x(odd number)  E.g 27 39 45 75 81 135 165 195 243 495...
mlmapping_lm_d_xyz:           0.1
mlmapping_lm_n_xy:            135
mlmapping_lm_n_z:             75
mlmapping_lm_log_odds_min:      -2.0
mlmapping_lm_log_odds_max:       4.0
mlmapping_lm_measurement_hit:    0.7
mlmapping_lm_measurement_miss:  -0.4
mlmapping_lm_occupied_sh:        2.2


use_exactsync:     false
mlmapping_sample_cnt:       500
mlmapping_use_raycasting:   true
mlmapping_enable_esdfs:     false
mlmapping_enable_2d_grid:   false

publish_T_wb:      true
publish_T_bs:      true

##transformation from sensor frame to body frame
sensor_frame_id:   "camera_depth_optical_frame"
T_B_S:
[  0.0,   0.0,  1.0,  0.0,
  -1.0,   0.0,  0.0,  0.0,
   0.0,  -1.0,  0.0,  0.0,
   0.0,   0.0,  0.0,  1.0]


body_frame_id:      "body_frame"
awareness_frame_id: "awareness_frame"
local_frame_id:     "local_frame"
world_frame_id:     "map"







